src/types.mo

```diff
@@ -642,20 +659,25 @@ module {
         return true;
       };
       case(#Map(x), #Map(y)){
```

Comment:
Why did you change Map as Set comparison? if I am not mistaken, the logic remains the same (does not take order into account), but it now uses sort which is O(n**2) for motoko algorithm if I remember correctly

Reply:
Previously the algo would take insertion order into account.  I put sort in so I could compare the items one for one.  I guess I could create a map not use sort.   I could also use get to make sure the item I'm inspecting in x is in y, but I think you said I shouldn't use get.

Reply:
I am a bit confused. You was previously looping through mapX and getting equivalent values from mapY to compare (meaning they could be at any position in mapY - not order dependent). Now you pre-sort 2 maps and compare 2 running iterators, which, as far as I understand, concludes to the same overall logic

Reply:
Previously you said to not use GET, so I was trying something different, if you think looping through x and Map.getting y is better I will revert.  I think that is better than sorting.